qstnrs/pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = qstnrs.settings
+addopts = --reuse-db
```

Comment:
--reuse-db is required because I don't think it's necessary to create the test database every time the tests run.